core-library/src/main/java/org/silverpeas/core/admin/persistence/SpaceUserRoleTable.java

```diff
@@ -234,153 +237,147 @@ public void removeSpaceUserRole(int id) throws AdminPersistenceException {
     updateRelation(DELETE_SPACEUSERROLE, id);
   }
 
-  static final private String DELETE_SPACEUSERROLE = "delete from ST_SpaceUserRole where id = ?";
+  private static final String DELETE_SPACEUSERROLE = "delete from ST_SpaceUserRole where id = ?";
 
   /**
    * Tests if a user has a given role (not recursive).
    */
   public boolean isUserDirectlyInRole(int userId, int spaceUserRoleId) throws
-      AdminPersistenceException {
+      SQLException {
     int[] ids = new int[] { userId, spaceUserRoleId };
     Integer result = getInteger(SELECT_COUNT_SPACEUSERROLE_USER_REL, ids);
 
     return result != null && result >= 1;
   }
 
-  static final private String SELECT_COUNT_SPACEUSERROLE_USER_REL = "select count(*) from "
+  private static final String SELECT_COUNT_SPACEUSERROLE_USER_REL = "select count(*) from "
       + "ST_SpaceUserRole_User_Rel where userId = ? and spaceUserRoleId = ?";
 
   /**
    * Add an user in a spaceUserRole.
    */
   public void addUserInSpaceUserRole(int userId, int spaceUserRoleId) throws
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Refactor this method to throw at most one checked exception instead of: java.sql.SQLException, org.silverpeas.core.admin.service.AdminException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1160)